src/reference-designs/tkg-edge.md

```diff
@@ -0,0 +1,885 @@
+
+For those companies that already have remote offices and branch offices with a few servers running edge workloads, running modern, distributed applications across your edge locations can add a new layer of complexity. There is a significant difference between running a cloud native platform at the core data center versus running it at edge locations.
+
+The VMware Edge Compute Stack helps companies manage their edge locations with little or no local IT staff and enables them to run VM workloads at these sites. Now with the addition of VMware Tanzu, you can simplify how you architect and deploy a cloud native stack at hundreds or thousands of edge locations so that you can run cloud native applications at the edge.
+
+The Tanzu Design for VMware Edge Compute Stack described here offers a best practice for running Tanzu at edge locations where the VMware Edge Compute Stack is already deployed. We’ll specifically address these challenges at the edge:
+
+- Running and managing Kubernetes infrastructure
+- Providing a consistent development and deployment environment for applications across the cloud, datacenter, and edge
+- Managing a fleet of Kubernetes clusters
+- Proactively monitoring clusters and apps
+- Ensuring security and compliance of edge environments.
+
+On top of the VMware Edge Compute Stack, the VMware Tanzu Edge Solution Architecture comprises core VMware Tanzu capabilities, including:
+
+- Unified Kubernetes Runtime: VMware Tanzu Kubernetes Grid (TKG) provides a consistent, upstream-compatible implementation of Kubernetes, that is tested, signed, and supported by VMware.
+- Global Multi-cluster Management: VMware Tanzu Mission Control (TMC) is a centralized management platform for consistently operating and securing your Kubernetes infrastructure and modern applications across multiple teams and clouds, regardless of where they reside.
+- Full-stack Observability: VMware Tanzu Observability by Wavefront (TO) is a high-performance streaming analytics platform that supports 3D observability (metrics, histograms, traces/spans). You can collect data from many services and sources across your entire application stack, including edge sites.
+
+This solution architecture is intended for anyone responsible for designing, implementing, managing and securing edge locations, including DevOps admins, VI admins, platform operators, and security teams.
+
+## Topology
+
+There are multiple topologies that can be leveraged for edge deployments. A centralized model, edge site are managed from a central hub (Hub and Spoke topology). A decentralized model is also available where each edge site is managed locally.
+This solution guide is focused on the centralized model where a vCenter is deployed at the main datacenter location and a datacenter with vSphere cluster at each edge location. It is assumed that the vSphere infrastructure is already deployed.
```

Comment:
Replace lines 27 and 28 with the following - 
Multiple topologies are possible for edge deployments. Some of the topologies are described in this section. However, the VMware Tanzu Edge Solution Architecture uses a centralized or hub and spoke topology, where a vCenter is deployed at the main datacenter location and a vSphere cluster is deployed in the datacenter at each edge location. This design document assumes that the vSphere infrastructure is already deployed.